option optimize_for = LITE_RUNTIME;

message UniteSkillShowNotify
{
 repeated uint32 sheng_jie_code = 1;
}
